src/DotNet.Status.Web/Models/AnnotationEntry.cs

```diff
@@ -0,0 +1,35 @@
+namespace DotNet.Status.Web.Models
+{
+    public class AnnotationEntry
+    {
+        /// <summary>
+        /// The original annotation sent from Grafana.
+        /// </summary>
+        public Annotation Annotation { get; set; }
+        /// <summary>
+        /// Time since UNIX Epoch in milliseconds. (required)
+        /// </summary>
+        public long Time { get; set; }
+        public long? TimeEnd { get; set; }
+        public bool? IsRange { get; set; } = false;
```

Comment:
Why nullable... but then with a default value of false? When is it actually ever set to null?

Reply:
I took this from the Grafana source, just trying to get the structure accurate. I can simplify it now that it's working properly.